services/spar/package.yaml

```diff
@@ -9,7 +11,10 @@ license-file:  LICENSE
 category:      System
 description:   'See README.md'
 
-ghc-options: -j -O2 -Wall -Wno-redundant-constraints -fwarn-tabs -Werror -optP-Wno-nonportable-include-path
+ghc-options:
+- -j
+- -Wno-redundant-constraints
+- -Werror
```

Comment:
Can you motivate the change (removal) of ghc options here?

Reply:
As Artyom mentioned, several of the more pervasive options are now globally applied via `module-defaults.yaml`